sorc/fre-nctools.fd/tools/fregrid/fregrid.c

```diff
@@ -15,10 +15,10 @@
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.
  
-  For the full text of the GNU General Public License,
-  write to: Free Software Foundation, Inc.,
-            675 Mass Ave, Cambridge, MA 02139, USA.  
------------------------------------------------------------------------
+  For the full text of the GNU General Public License, write to:
+
+  Free Software Foundation, Inc.,
+  675 Mass Ave, Cambridge, MA 02139, USA.  
```

Comment:
Take out the long license text. We already have it in the README.

Reply:
Done.